### PR TITLE
Change groups for testing

### PR DESCRIPTION
**Pull request by AlexChetverov**

### CODE REVIEW

**Review by mateuszaaa — APPROVED** [review]
lgtm

File: .github/workflows/reusable-e2e-tests.yml [hunk 1]
```diff
@@ -248,33 +252,34 @@ jobs:
...
         if: failure()
         run: echo '[GCP Bucket link](https://console.cloud.google.com/storage/browser/mangata-node-ci-cache/${{ github.sha }}/${{ env.MATRIX_COMMAND_PATH }})' >> $GITHUB_STEP_SUMMARY
 
-      - name: Stop previous parachain if running
-        working-directory: devops/parachain-launch/output
+      - name: Stop docker instance
+        if: always()
         run: |
           docker kill $(docker ps -q) 2>/dev/null && echo $?
           docker-compose down -v
```
Comment (tenequm) [inline .github/workflows/reusable-e2e-tests.yml]: you can remove this step, it doesn't make much sense, as everything will be automatically purged when job is completed

Comment (goncer) [reply .github/workflows/reusable-e2e-tests.yml]: removed!

**Review by tenequm — APPROVED** [review]
lgtm

**Pull request merged**